#! /usr/bin/env python

PACKAGE='peter_chargingpile_search'

#from driver_base.msg import SensorLevels
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("break_distance", double_t, 0, "break_distance unit:m", 0.02, 0.0, 0.04)
gen.add("ignore_point_num", int_t, 0, "ignore_point_num unit:none", 30, 1, 50)
gen.add("max_salient_tolerance", double_t, 0, "max_salient_tolerance unit:m", 0.0174, 0.0, 0.03)
gen.add("max_variance_tolerance", double_t, 0, "max_variance_tolerance unit:Standard Deviation", 0.01, 0.0, 0.02)
gen.add("max_festureangle_tolerance", double_t, 0, "max_festureangle_tolerance unit:angle", 1.8, 0.0, 3.0)

gen.add("linear_vel", double_t, 0, "linear_vel unit:none", 0.15, 0.0, 1.0)
gen.add("angular_vel", double_t, 0, "angular_vel unit:none", 0.5, 0.0, 3.0)



exit(gen.generate(PACKAGE, "Config", "peterChargingNodeParams"))

